app/src/main/res/layout/screen_addidtopatientsearch.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="utf-8"?>
+<org.simple.clinic.addidtopatient.searchforpatient.AddIdToPatientSearchScreen android:id="@+id/patientsearch_root"
+  xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  xmlns:tools="http://schemas.android.com/tools"
+  android:layout_width="match_parent"
+  android:layout_height="match_parent">
+
+  <androidx.appcompat.widget.Toolbar
+    android:id="@+id/addidtopatientsearch_toolbar"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    app:navigationIcon="@drawable/ic_arrow_back_24dp"
+    tools:ignore="ContentDescription">
+
+    <TextView
+      android:id="@+id/addidtopatientsearch_title"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:layout_marginBottom="@dimen/spacing_16"
+      android:layout_marginTop="@dimen/spacing_16"
+      android:ellipsize="end"
+      android:maxLines="2"
+      android:text="@string/addidtopatientsearch_title"
+      android:textAppearance="@style/Clinic.V2.TextAppearance.Body0Left.White72" />
+
+  </androidx.appcompat.widget.Toolbar>
+
+  <ScrollView
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:layout_above="@+id/addidtopatientsearch_search"
+    android:layout_below="@id/addidtopatientsearch_toolbar"
+    android:scrollbarStyle="insideOverlay"
+    android:scrollbars="vertical">
+
+    <LinearLayout
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:orientation="vertical"
+      android:paddingBottom="64dp"
+      android:paddingEnd="@dimen/spacing_20"
+      android:paddingStart="@dimen/spacing_24"
+      android:paddingTop="@dimen/spacing_16"
```

Comment:
The `paddingTop` is `24dp` as per the Zeplin designs.

Reply:
Fixed.